### ISSUE TITLE
For all three-radio button of Network Concurrency group NVDA is announcing as one of one: A11y_Azure Tools Storage Explorer_Settings_Transfers_Screen Reader

### ISSUE THREAD

**Issue by kapilvaishna** [microsoft/AzureStorageExplorer#8376]
### Preflight Checklist

- [x] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [x] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [x] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.36.2

### Architecture

x64

### Platform

Windows

### OS Version

Windows11

## GitHub Tags
#A11yMAS; #A11yTCS; #Win32; #DesktopApp; #A11ySev2; #BM_AzureToolsStorageExplorer_Win32_Dec2024; #Azure Tools Storage Explorer; #Screenreader; #WCAG1.3.1; #FTP; #NVDA; #Jaws; #8407 

## Environment Details:
Application: Microsoft azure storage explorer
Version: 1.36.2
OS: Windows 11 Enterprise 24H2
Build: 26100.2605
NVDA Version: 2024.4.1
JAWS Version: 2024.2312.53

## Repro Steps

1. Turn on NVDA
2. Launch the Azure Tools Storage Explorer.
3. TAB till "Setting" in left navigation and press ENTER key. 
4. TAB till "Transfer" and press ENTER key.
5. TAB till Network Concurrency radio group.
6. Observe that for all three-radio button of Network Concurrency group NVDA is announcing as one of one.

## Actual Experience

For all three-radio button of Network Concurrency group NVDA is announcing as one of one.
Note: Issue is repro with JAWS also

## Expected Experience

NVDA should announce correct numeric grouping for radio buttons.

##User Impact:
Screen reader get confused as there three radio buttons are there but screen reader announces as one of one. 

## Attachment
NVDA
![Image](https://github.com/user-attachments/assets/529de41c-75ad-4bb1-add2-1cc78f829616)

Jaws
https://github.com/user-attachments/assets/c7c818bf-59f4-4ded-a67a-47676431a154

**Comment by craxal** [comment]
HTML in there definitely needs some work.

**Comment by InduPriya1805** [comment]
Rev:inse

**Comment by craxal** [comment]
@kapilvaishna Fix has been merged. Please verify with new build.

**Comment by kapilvaishna** [comment]
Issue is repro in 1.37.0 version, please find repro attachment below

![Image](https://github.com/user-attachments/assets/300ecbeb-6f04-42b9-bab5-2bee0d79975a)

![Image](https://github.com/user-attachments/assets/28d46353-a1cd-4928-a768-3c5a0b359c23)

**Comment by craxal** [comment]
@kapilvaishna Which branch did you get that build from? You should be getting builds from the main branch.

**Comment by kapilvaishna** [comment]
Issue is not repro in below build so closing the bug.

Application: Microsoft azure storage explorer
Version: 1.37.0 (104)


![Image](https://github.com/user-attachments/assets/a011143b-7aea-4897-9804-b2a646fe0559)
